### ISSUE TITLE
macOS Mojave SDL2 Rendering

### ISSUE THREAD

**Issue by abowerman** [tomassedovic/roguelike-tutorial#35]
Just a heads up if you're having issues on Mojave

https://github.com/tomassedovic/tcod-rs/issues/267#issuecomment-443960544

**Comment by abowerman** [comment]
Issues like a blank grey screen, etc..

**Comment by yngtodd** [comment]
Thanks @abowerman for linking that issue. That helped me get up and running. 

You can find the fix in the linked thread above, but, if anyone finds their way here, here is the TLDR that worked for me:

Brew install of sdl12:
```
brew install pkg-config sdl2
```

Cargo.toml:
```
 [dependencies]
 tcod = { git = "https://github.com/rjungemann/tcod-rs.git", branch = "fix-sdl2-rendering-mojave" }
```

My system:
```
2013 Macbook Air
macOS Mojave version 10.14.3 (18D109)
```

**Comment by tomassedovic** [comment]
That branch has been merged and published in tcod-rs 0.15.0.

The tutorial has been updated to use that version. I'm going to close this. Please reopen if you still see this even with the latest tutorial & tcod.
